src/components/Users/UserAvailabilityTab.tsx

```diff
@@ -107,7 +128,12 @@ export default function UserAvailabilityTab({
       <Calendar
         className="lg:order-last"
         month={month}
-        onMonthChange={setMonth}
+        onMonthChange={(newMonth) => {
+          setMonth(newMonth);
+          const start = format(startOfMonth(newMonth), "yyyy-MM-dd'T'HH:mm");
+          const end = format(endOfMonth(newMonth), "yyyy-MM-dd'T'HH:mm");
+          setQParams({ valid_from: start, valid_to: end });
```

Comment:
let's not store this in query params. we can derive the `valid_from` and `valid_to` filter values from the existing `month` state.